docker-compose.yml

```diff
@@ -38,13 +38,6 @@ services:
     restart: always
     ports:
       - "1883:1883"
-      - "8883:8883"
```

Comment:
Why is the port 8883 removed?

Reply:
Without the certificates this port (TLS mqtt) won´t works